### PR TITLE
Feat: Image and Input

### PR DESCRIPTION
**Pull request by Sen-442b**
**Features: -**
- Responsive images 
-  Form input with validation texts  with simple username, email and password verification conditions 
- Textbox with three different size choices

**Limitations: -**
-  Repetitive JS code
- Verbose Markup 
- Poor client-side form authentication 

**Upcoming Features**

- Less html validation elements (probably by using custom attributes)
- Strong validation features like complex email and phone number validation

### CODE REVIEW

**Comment by Sen-442b** [comment]
[Watch the video](https://youtu.be/jo0PQHlBlxo)

File: Image/source/image.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.image-container{
+    width: 30rem;
+    
+
```
Comment (Swapnilkota108) [inline Image/source/image.css]: you can remove the empty lines

**Review by Swapnilkota108 — CHANGES_REQUESTED** [review]

File: Image/source/image.css (hunk 1 above)
Comment (Sen-442b) [reply Image/source/image.css]: Updated,  kindly check it whenever you're free

File: Image/source/image.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
+.image-container {
```
Comment (msdeshmukh009) [inline Image/source/image.css:1]: Consider keeping these styles in the different stylesheet as these are more related to the documentation site if I am not wrong.

File: Input/source/input.css [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  padding: 0.5em;
+}
+
+.display-none {
+  display: none;
+}
+
+.display-block {
+  display: block;
+}
```
Comment (msdeshmukh009) [inline Input/source/input.css:95]: `display-none` & `display-block` can be used in many places consider shifting these classes to the global stylesheet in the future maybe.

**Review by msdeshmukh009 — APPROVED** [review]

**Pull request merged**